plugins/pulp_rpm/plugins/importers/yum/pulp_solv.py

```diff
@@ -748,7 +696,12 @@ def add_repo_units(self, units, repo_id, installed=False):
 
         for unit in units:
             try:
-                factory = self.type_factory_mapping[unit['_content_type_id']]
+                type_factory_mapping = {
```

Comment:
It looks like a constant to me. Any reason to move this from the class attribute?
Maybe worth putting outside of the `for` loop and `try-except`? or maybe I just miss something...

Reply:
I probably just moved it temporarily and committed it accidentally.